composer.json

```diff
@@ -9,7 +9,7 @@
     ],
     "minimum-stability": "stable",
     "require-dev": {
-        "phpunit/phpunit": "3.7.*",
+        "phpunit/phpunit": "4.6.*",
```

Comment:
You can still use the one from the composer if it's there. It'll be updated more frequently than travis', and it would make sense to use the same version on local machines as on the ci.